ekf: {
x0: [0,  # PX
     0,  # PY
     0,  # PZ
     0,  # U
     0,  # V
     0,  # W
     0,  # PHI
     0,  # THETA
     0,  # PSI
     0,  # AX
     0,  # AY
     0,  # AZ
     0,  # BX
     0,  # BY
     0], # BZ
P0: [0.005,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PX
     0.005,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PY
     0.005,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PZ
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # U
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # V
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # W
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PHI
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # THETA
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PSI
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AX
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AY
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AZ
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BX
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BY
     0.020],                                 # BZ

Q0: [0.005,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PX
     0.005,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PY
     0.005,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PZ
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # U
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # V
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # W
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PHI
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # THETA
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PSI
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AX
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AY
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AZ
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BX
     0.020,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BY
     0.020],                                 # BZ

R_IMU: [0.05,  0,0,0,
        0.05,  0,0,0,
        0.05],

R_Mocap: [0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01],

inner_loop_rate: 200,
alpha: 0.3
}
